pkg/rc/replication_controller.go

```diff
@@ -196,6 +197,11 @@ func (rc *replicationController) WatchDesires(quit <-chan struct{}) <-chan error
 	errOutChannel := make(chan error)
 	channelsClosed := make(chan struct{})
 
+	// get random wait time to check for missing artifacts bewteen 1 - 60 minutes
+	randomTime := rand.Intn(60) + 1
+	shouldInitialCheck := time.After(fmt.Sprintf("%dm", randomTime))
+	shouldPeriodicCheck := time.Tick(fmt.Sprintf("%dm", randomTime+60))
```

Comment:
i'd probably make this a `time.NewTimer(randomTime * time.Minute + time.Hour)` to start off with and then call `timer.Reset(time.Hour)` each time it fires, that way we get exactly hour spacing after the random time

Reply:
what you have is fine too, it's just weird for there to be a random initialization time AND a random interval after that, but maybe i'm being too particular

Reply:
Oops, I meant something like that, but what you said can inspired me to do this with one timer.